Fig.Core/Settings.cs

```diff
@@ -60,6 +60,27 @@ public bool Update(object val)
             }
         }
 
+        private interface IBindResult
+        {
+            IEnumerable<string> Errors { get; }
+            object Result { get; }
+        }
+
+        private class BindResult<T> : IBindResult
+        {
+            public BindResult(List<string> errors, T result)
+            {
+                Errors = errors;
+                Result = result;
+            }
+            public List<string> Errors { get; }
+            public T Result { get; }
+
+            IEnumerable<string> IBindResult.Errors => this.Errors;
```

Comment:
Likewise curious about the the explicit implementation as opposed to "regular" implicit implementation

Reply:
See my comment above.  It does seem kinda hacky so if there is a better way, I'm all for it.